### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S deps` and `typos --format brief`

### CODE REVIEW

File: lib/bumblebee.ex [hunk 1]
```diff
@@ -607,7 +607,7 @@ defmodule Bumblebee do
   of subsequent timesteps for model forward pass.
 
   Note that the number of `timesteps` may not match `num_steps` exactly.
-  `num_steps` parameterizes sampling points, however depending on the
```
Comment (seanmor5) [inline lib/bumblebee.ex:610]: I think both of these spellings are actually acceptable

File: lib/bumblebee.ex [hunk 2]
```diff
@@ -607,7 +607,7 @@ defmodule Bumblebee do
...
+  `num_steps` parametrizes sampling points, however depending on the
```
Comment (jonatanklosko) [inline lib/bumblebee.ex:610]:
```suggestion
  `num_steps` parameterizes sampling points, however depending on the
```

**Review by jonatanklosko — APPROVED** [review]
Thanks :)

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**